### ISSUE TITLE
ValueError: not enough values to unpack (expected 2, got 1)

### ISSUE THREAD

**Issue by soichih** [yeatmanlab/AFQ-Browser#270]
I am seeing following error message when I try to run afqbrowser-assemble.

```
Traceback (most recent call last):
  File "/usr/local/bin/afqbrowser-assemble", line 49, in <module>
    link=args.page_title_link, sublink=args.page_subtitle_link)
  File "/usr/local/lib/python3.6/site-packages/afqbrowser/browser.py", line 485, in assemble
    out_path=out_path)
  File "/usr/local/lib/python3.6/site-packages/afqbrowser/browser.py", line 258, in afq_mat2tables
    n_subjects, nodes_per_tract = vals[stats[0]].item()[0].shape
ValueError: not enough values to unpack (expected 2, got 1)
```

Here is the AFQ .mat file I am using 
> https://www.dropbox.com/s/1n0sgvvx2eklrs2/output.mat

I've created this output with following code

```
config.dtiinit  = '/mnt/scratch/testdata/dtiinit2';
sub_dirs = { [config.dtiinit '/dti'] };
sub_group = [1]; %0 = control 1 = patient
afq = AFQ_Create('run_mode', 'test', 'sub_dirs', sub_dirs, 'sub_group', sub_group, 'showfigs', 0);
[afq patient_data control_data norms abn abnTracts] = AFQ_run(sub_dirs, sub_group, afq);
save('output.mat', 'afq');
```

Maybe I should set sub_gruop to 0?

**Comment by arokem** [comment]
Do I understand correctly: you have only one subject in that matfile? I
think we don't currently cover that use-case, but maybe we should.

**Comment by soichih** [comment]
Yes, I am only passing 1 subject right now. I am trying to use AFQbrowser to visualize a single subject's tract profiles. If that's not what AFQ browser is supposed to do, then I can probably pass some static data from another subject as a control, and use it to visualize the input subject (as a patient?) against? Would you recommend that route?

**Comment by arokem** [comment]
Turns out it wasn't too hard to make it work for a single subject. See #271 and screenshot below: 
<img width="1791" alt="screen shot 2018-04-12 at 12 16 07 pm" src="https://user-images.githubusercontent.com/118582/38698892-6050b59e-3e4b-11e8-8a62-181d0aaf0e88.png">

**Comment by soichih** [comment]
nice!

**Comment by arokem** [comment]
Closed through #271
